IaC/pmm.cd/JenkinsStack.yml

```diff
@@ -810,6 +830,21 @@ Resources:
                 systemctl reload sshd
               }
 
+              setup_ssh_keys() {
+                KEYS_LIST="alexander.tymchuk mykola.marzhan evgeniy.patlan slava.sarzhan illia.pshonkin alex.miroshnychenko eduardo.casarero santiago.ruiz andrew.siemen kamil.babayev"
+
+                for KEY in $KEYS_LIST; do
+                    if [[ ! -f "/tmp/$KEY.pub" ]]; then
+                        until wget -O "/tmp/$KEY.pub" "https://www.percona.com/get/engineer/KEY/$KEY.pub"; do
```

Comment:
I think we should let it fail in case a person left the company and you are not aware or have not updated this file.

Reply:
We must not remove this conditional check

But you're right, we should improve it, if it returns 404, it should be skipped.